.travis.yml

```diff
@@ -15,24 +15,18 @@ script:
 matrix:
   fast_finish: true
   include:
-  - rvm: 2.1.9
```

Comment:
Should we still test against Ruby 2.9.1? I thought Puppet 4.7.0 is still built off that. If so, I think we should leave ruby 2.1.9 in our test matrix for Puppet 4.x tests

Reply:
Good one, that's IMHO something that should be fixed in the modulesync repo.

Reply:
I'll go ahead and merge this, but I agree, we need to change this in the modulesync repo